src/CacheTool/Adapter/Cli.php

```diff
@@ -24,7 +24,11 @@ protected function doRun(Code $code)
         $file = $this->createTemporaryFile();
         $code->writeTo($file);
 
-        $process = new Process("php $file");
+        $php = 'php';
+        if (defined('PHP_BINARY')) {
```

Comment:
Is this check necessary? When is `PHP_BINARY` not defined? The docs, https://www.php.net/manual/en/reserved.constants.php, lead me to believe it is always available. Do you have an example when this wouldn't be available?